src/typings.ts

```diff
@@ -0,0 +1,286 @@
+/**
+ * Discord ID
+ */
+type Snowflake = string
+
+interface BotInfo {
+  /**
+   * The id of the bot
+   */
+  id: Snowflake
+  /**
+   * The username of the bot
+   */
+  username: string
+  /**
+   * The discriminator of the bot
+   */
+  discriminator: string
+  /**
+   * The avatar hash of the bot's avatar
+   */
+  avatar?: string
+  /**
+   * The cdn hash of the bot's avatar if the bot has none
+   */
+  defAvatar: string
+  /**
+   * The library of the bot
+   */
+  lib: string
+  /**
+   * The prefix of the bot
+   */
+  prefix: string
+  /**
+   * The short description of the bot
+   */
+  shortdesc: string
+  /**
+   * The long description of the bot. Can contain HTML and/or Markdown
+   */
+  longdesc: string
+  /**
+   * The tags of the bot
+   */
+  tags: Array<string>
+  /**
+   * The website url of the bot
+   */
+  website?: string
+  /**
+   * The support server invite code of the bot
+   */
+  support?: string
+  /**
+   * The link to the github repo of the bot
+   */
+  github?: string
+  /**
+   * The owners of the bot. First one in the array is the main owner
+   */
+  owners: Array<Snowflake>
+  /**
+   * The guilds featured on the bot page
+   */
+  guilds: Array<Snowflake>
+  /**
+   * The custom bot invite url of the bot
+   */
+  invite?: string
+  /**
+   * The date when the bot was approved (in ISO 8601)
+   */
+  date: string
+  /**
+   * The certified status of the bot
+   */
+  certifiedBot: boolean
+  /**
+   * 	The vanity url of the bot
+   */
+  vanity?: string
+  /**
+   * The amount of upvotes the bot has
+   */
+  points: number
+  /**
+   * The amount of upvotes the bot has this month
+   */
+  monthlyPoints: number
+  /**
+   * The guild id for the donatebot setup
+   */
+  donatebotguildid: Snowflake
+}
+
+interface BotStats {
+  /**
+   * The amount of servers the bot is in
+   */
+  server_count?: number
+  /**
+   * The amount of servers the bot is in per shard. Always present but can be empty
+   */
+  shards: Array<number>
+  /**
+   * The amount of shards a bot has
+   */
+  shard_count?: number
+}
```

Comment:
We're going to be changing the inconsistency issues with casing in v2, can the response from the v1 api be transformed to fit consistent casing so the wrapper doesn't look inconsistent when we get to v2?
```suggestion
interface BotStats {
  /**
   * The amount of servers the bot is in
   */
  serverCount?: number
  /**
   * The amount of servers the bot is in per shard. Always present but can be empty
   */
  shards: number[]
  /**
   * The amount of shards a bot has
   */
  shardCount?: number
}
```